src/schemas/json/cryproj.52.schema.json

```diff
@@ -489,7 +489,7 @@
 				"ai_DrawProbableTarget",
 				"ai_DrawRadar",
 				"ai_DrawRadarDist",
-				"ai_DrawReadibilities",
+				"ai_DrawReadabilities",
```

Comment:
This appears to be the [official](https://docs.cryengine.com/display/CRYAUTOGEN/CONSOLEPREFIXAI#AnchorAIDRAWREADIBILITIES) (misspelled) term.